modules/appeals_api/app/controllers/appeals_api/docs/v2/docs_controller.rb

```diff
@@ -4,9 +4,7 @@ class AppealsApi::Docs::V2::DocsController < ApplicationController
   skip_before_action(:authenticate)
 
   def decision_reviews
-    filename = Settings.vsp_environment == 'production' ? 'swagger.json' : 'swagger_dev.json'
-    file = File.read AppealsApi::Engine.root.join("app/swagger/appeals_api/v2/#{filename}")
-    render json: JSON.parse(file)
+    render json: JSON.parse(swagger_file('decision_reviews', version: 'v2'))
```

Comment:
Leaving this comment here since I can't comment on the moved files below:

I changed the internal folder structure in the `app/swagger` directory so that specs are nested inside (1) a folder with the `ENV['API_NAME']` (so, 'higher_level_reviews', for example, and now 'decision_reviews' for decision reviews, etc.), and (2) the version (v0 or v2). This makes it easier to use one shared helper to generate paths as shown in the docs controller here.

(The actual behavior of the docs routes remains the same - they just load this json from a different place vs. before)